tutorials/scripting/gdextension/gdextension_c_example.rst

```diff
@@ -0,0 +1,2154 @@
+binding it. So we're gonna create a helper function that can call a MethodBind
+with these types. Even though it does return something (an error code), we don't
+need to deal with it, so for now we're just going to ignore it.
+
+In the ``api.h``, we're adding a few things to the existing structs, plus a new
+helper function for the call:
+
+.. code-block:: c
+
+    struct Constructors
+    {
+        ...
+        GDExtensionVariantFromTypeConstructorFunc variant_from_string_name_constructor;
+        GDExtensionVariantFromTypeConstructorFunc variant_from_vector2_constructor;
+    } constructors;
+
+    struct Destructors
+    {
+        ..
+        GDExtensionInterfaceVariantDestroy variant_destroy;
+    } destructors;
+
+    ...
+
+    struct Methods
+    {
+        ...
+        GDExtensionMethodBindPtr object_emit_signal;
+    } methods;
+
+    struct API
+    {
+        ...
+        GDExtensionInterfaceObjectMethodBindCall object_method_bind_call;
+    } api;
+
+    ...
+
+    // Helper to call with Variant arguments.
+    void call_2_args_stringname_vector2_no_ret_variant(
+        GDExtensionMethodBindPtr p_method_bind,
+        GDExtensionObjectPtr p_instance,
+        const GDExtensionTypePtr p_arg1,
+        const GDExtensionTypePtr p_arg2);
+
+Now let's switch to the ``api.c`` file to load these new function pointers and
+implement the helper function.
+
+.. code-block:: c
+
+    void load_api(GDExtensionInterfaceGetProcAddress p_get_proc_address)
+    {
+        // API.
+        ...
+        api.object_method_bind_call = p_get_proc_address("object_method_bind_call");
+
+        // Constructors.
+        ...
+        constructors.variant_from_string_name_constructor = api.get_variant_from_type_constructor(GDEXTENSION_VARIANT_TYPE_STRING_NAME);
+        constructors.variant_from_vector2_constructor = api.get_variant_from_type_constructor(GDEXTENSION_VARIANT_TYPE_VECTOR2);
+
+        // Destructors.
+        ...
+        destructors.variant_destroy = p_get_proc_address("variant_destroy");
+
+        ...
+    }
+
+    ...
+
+    void call_2_args_stringname_vector2_no_ret_variant(GDExtensionMethodBindPtr p_method_bind, GDExtensionObjectPtr p_instance, const GDExtensionTypePtr p_arg1, const GDExtensionTypePtr p_arg2)
+    {
+        // Set up the arguments for the call.
+        Variant arg1;
+        constructors.variant_from_string_name_constructor(&arg1, p_arg1);
+        Variant arg2;
+        constructors.variant_from_vector2_constructor(&arg2, p_arg2);
+        GDExtensionConstVariantPtr args[] = {&arg1, &arg2};
+
+        // Add dummy return value storage.
+        Variant ret;
+
+        // Call the function.
+        api.object_method_bind_call(p_method_bind, p_instance, args, 2, &ret, NULL);
+
+        // Destroy the arguments that need it.
+        destructors.variant_destroy(&arg1);
+        destructors.variant_destroy(&ret);
+    }
+
+This helper has some boilerplate but is quite straightforward. It sets up the
```

Comment:
```suggestion
This helper function has some boilerplate code but is quite straightforward. It sets up the
```

Reply:
``quite straightforward`` should also be avoided as it sounds like it's trivial to understand. Perhaps just end the sentence after ``boilerplate``